apps/projects/app/components/App/App.js

```diff
@@ -28,7 +29,7 @@ let CLIENT_ID = ''
 let REDIRECT_URI = ''
 let AUTH_URI = ''
 
-let ipfs = ipfsClient({ host: 'localhost', port: '5001', protocol: 'http' })
+let infuraIpfs = ipfsClient({ host: 'ipfs.infura.io', port: '5001', protocol: 'https' })
```

Comment:
Long term we should migrate to using the new aragon.js ipfs utilities but this approach is fine for now.

Reply:
aragon.js has utilities that pins to ipfs?

Reply:
Yeah long term there are a lot of cool things we can do, even use the dweb loader: https://github.com/DistributedWeb/dweb (probably after filecoin gets established). For now i think this is fine